### DISCUSSION TITLE
What are the good Web3 resources that you recommend to your friends?

### DISCUSSION THREAD

**Discussion by samnang** [web3together/web3together#16] [General]
Web3 is quite a new space to explore. There are many topics, resources, and people to follow. Sometimes this information is too overwhelming. From your journey in Web3, what are people that you learn from or good resources that you find them useful to recommend to others people to follow.

Note: Web3Together collects and compiles a [list of resources](https://github.com/web3together/web3together), so others will find them useful. What do you want to add there?

**Comment by saingsab** [comment]
This is not self promoted, I want to share part of my AMA discussion with @jensendarren about Web3 and Blockchain. 
Full playlist : https://youtube.com/playlist?list=PLjhxEYHz6zkdcSxQv9ULwnimRjcWKC9l5 

Web revlution from web1.0 to web2.0 and Web3.0 : https://youtu.be/bNw-bTHZo7k?t=669

[Spoil alert not my favorite show yet haha].

**Reply by samnang** [reply]
Great, love to see more content that we can create to share with our members 👍

**Reply by samnang** [reply]
@saingsab thanks for the suggestion, I have added under the links section here https://github.com/web3together/web3together#links

**Comment by borinmin** [comment]
I joined the session last week, it was a good discussion. I don't know how much that participants could grab the concepts of the smart contract applications, as a listener, I am very fresh to smart contract was very tough to catch the smart contract's goals and when bong Bunhouth was demonstrating, I'm totally blank about smart contract's processes. Anyway, thank you for the great content.

**Reply by samnang** [reply]
Thanks for your feedbacks and great to hear you find the content is useful.

You are not alone, Web3 space is quite confusing topic because there is no real definition, each individual tries to come up with their own definition. There are not so many dapps that solve real pratical use case in the real world yet, so that's why for new people in the space is hardly understanding how it works and even harder for non-tech people to get started in this technology, but it's getting better and better because more and more people start involving, educating, and building solution.

Don't forget we are a small community here, feel free to raise/ask any question or feedbacks from your journey. Some members have been a while in the space, so they can share their experiences in the community.

**Reply by borinmin** [reply]
Thank you bong!

**Comment by csokun** [comment]
Once you get a hang of creating, deploy & interacting with Smart Contract. Next tool you should get familiar with https://docs.openzeppelin.com/cli

**Reply by samnang** [reply]
Thanks for the suggestions, OpenZeppelin seems to be the goto for token implementations right now.

**Reply by csokun** [reply]
I'm less interested in Token, here is the cool feature I can't leave without https://docs.openzeppelin.com/cli/2.8/getting-started#upgrading-your-contract
